R/run_pacta.R

```diff
@@ -31,6 +39,10 @@ run_pacta <- function(config) {
 
   dir.create(output_path_standard, recursive = TRUE, showWarnings = FALSE)
 
+  by_group <- get_aggregate_alignment_metric_by_group(config)
+  by_group <- check_and_prepare_by_group(by_group)
+  by_group_ext <- if (is.null(by_group)) { "_meta" } else {paste0("_", by_group, collapse = "_")}
```

Comment:
nit: I'm embarrassed to admit, I'm not sure what `paste0("_", by_group, collapse = "_")` will do without trying it. Is `collapse = "_"` basically ignored because both of the objects being pasted have length 1? If so, and that's always the case, maybe remove `collapse = "_"`?
```suggestion
  by_group_ext <- if (is.null(by_group)) { "_meta" } else { paste0("_", by_group) }
```

Reply:
that is correct